components/brave_wallet_ui/common/hooks/assets.test.ts

```diff
@@ -31,8 +31,6 @@ const mockVisibleList = [
   AccountAssetOptions[1]
 ]
 
-const expectedResult = [AccountAssetOptions[0]]
```

Comment:
Can we move this to the specific test where it is used?

Reply:
This was removed, no longer used.